amazon_payments/views.py

```diff
@@ -540,6 +543,11 @@ def post(self, request, *args, **kwargs):
             order_total = self.get_order_totals(
                 self.request.basket,
                 shipping_method=shipping_method)
+
+            request.basket.calculate_tax(
+                shipping_address
+            )
+            ###
```

Comment:
@sjolicoeur Can you comment on this as well? https://github.com/britco/django-oscar-amazon-payments/pull/1#discussion_r34180422